src/helpers/report-helpers.ts

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios'
+
+// eslint-disable-next-line import/no-cycle
+import { ReportsApiClientV20210630 } from '../api-clients/reports-api-client-v20210630'
+import { AMAZON_US_MARKETPLACE_ID } from './constants'
+import { sleep, tabDelimitedToArray } from './utils'
+
+interface ReportParameters {
+  startDate: string
+  endDate: string
+  marketplaceIds?: string[]
+}
+
+/**
+ * Allow other people to use the parsing function
+ */
+export { tabDelimitedToArray as parseAmazonReport }
```

Comment:
Why export this here and not on the index where everything else gets exported?

Reply:
moved to index